trillian/ctfe/ct_server/main.go

```diff
@@ -176,7 +177,7 @@ func main() {
 	// Register handlers for all the configured logs using the correct RPC
 	// client.
 	for _, c := range cfg.LogConfigs.Config {
-		inst, err := setupAndRegister(ctx, clientMap[c.LogBackendName], *rpcDeadline, c, corsMux, *handlerPrefix, *maskInternalErrors)
+		inst, err := setupAndRegister(ctx, clientMap[c.LogBackendName], *rpcDeadline, c, corsMux, *handlerPrefix, *maskInternalErrors, *revocationTreeID)
```

Comment:
it's about time we start making breaking changes, but its worth noting that this is a breaking change. i.e., we won't be compatible with the existing CT repo and may have to rewrite breaking tests.